### ISSUE TITLE
error at save with editAll/overrideAll

### ISSUE THREAD

**Issue by zonky2** [contao-community-alliance/dc-general#501]
editAll:
```mm_disease::1
No saved model[1]. The property "alias.dummy" is already in the database and not unique.
```

overrideAll:
```
The property "alias.dummy" is already in the database and not unique.
mm_disease::1
No saved model[1]. The property "alias.dummy" is already in the database and not unique.
```

**Comment by zonky2** [comment]
@baumannsven with DCG 2.1.2 works fine

**Comment by discordier** [comment]
So the problem is somewhere in here https://github.com/contao-community-alliance/dc-general/compare/2.1.2...2.1.3

**Comment by baumannsven** [comment]
Fixed in dd31e47680d465a0b265d1de08a62162ae131e6b
